src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -1,15 +1,19 @@
 package lotto.domain;
 
+import lotto.exception.LottoNumberSizeException;
+
+import java.math.BigDecimal;
 import java.util.*;
 
 import static lotto.domain.Number.MAX_NUMBER;
 import static lotto.domain.Number.MIN_NUMBER;
 import static lotto.utils.RandomNumberUtils.generateRandomNumbers;
+import static lotto.utils.ValidationUtils.*;
 
 public class LottoNumber {
 
     public static final int LOTTO_SIZE = 6;
-    public final static int GAME_PRICE = 1000;
+    public final static BigDecimal GAME_PRICE = BigDecimal.valueOf(1000);
```

Comment:
가격에 대한 부분이니 Money를 이용해볼 수 도 있지 않을까요 ? :)

```suggestion
    public final static Money GAME_PRICE = new Money(BigDecimal.valueOf(1000));
```